lib/AST/Module.cpp

```diff
@@ -1646,6 +1646,11 @@ LookupConformanceInModuleRequest::evaluate(
 
   auto nominal = type->getAnyNominal();
 
+  // If it's not a nominal, then it's safe to assume its copyable.
```

Comment:
Tuples should probably not be copyable unless all of their fields are copyable; is that already handled?

Reply:
yes tuples and all of the other kinds of types checked-for just before this are aware of Copyable. I didn't really know how to handle pack types / `repeat while` so I just stuck this here to catch "types not otherwise handled". I can clarify that in the comment.

Reply:
actually i'll just put specific handling of `PackExpansionType` for Copyable since that's the only reason I put this catch-all here.

Reply:
Conformance lookup with a PackExpansionType should not actually occur; it would be fine to put an assertion guarding against that.

As for a PackType, it will conform to Copyable if every element conforms to Copyable. The existing pack conformance logic should handle that because it recursively looks up the conformance on each element of the pack.